css/src/atomics/spacing.scss

```diff
@@ -96,3 +96,7 @@ $layout-sizes: (
 		}
 	}
 }
+
```

Comment:
`spacing.scss` is more about margins and paddings. I might recommend putting this in `atomics/typography.scss` instead?